metadata/metadata_test.go

```diff
@@ -29,13 +29,13 @@ const TEST_REPOSITORY_DATA = "../testutils/repository_data/repository/metadata"
 
 var fixedExpire = time.Date(2030, 8, 15, 14, 30, 45, 100, time.UTC)
 
-func getSignatureByKeyID(signatures []Signature, keyID string) HexBytes {
-	for _, sig := range signatures {
+func getSignatureByKeyID(signatures []Signature, keyID string) (HexBytes, int) {
```

Comment:
Why do you need the slice index?

Reply:
It's for [TestMetadataVerifyDelegate](https://github.com/rdimitrov/go-tuf-metadata/pull/54/files/97cd3616a9a36bc4f60d7a7b64d07aba11f95ba2#diff-febffd881a5f8de233564e404dab90eeb9e81e428dbba2c6e4c8665be7aba6acR496) . This allows verifying Signature per key ID.

Reply:
Got it. I see it's mainly used for understanding if it failed so can we avoid using the index of -1 for that and assert if the byte slice is empty instead?